internal/controllers/httpsedge_controller.go

```diff
@@ -195,6 +196,16 @@ func (r *HTTPSEdgeReconciler) reconcileRoutes(ctx context.Context, edge *ingress
 			return err
 		}
 
+		if routeSpec.IPRestriction != nil {
+			if err := routeModuleUpdater.ipPolicyResolver.validateIPPolicyNames(ctx, edge.Namespace, routeSpec.IPRestriction.IPPolicies); err != nil {
+				if apierrors.IsNotFound(err) {
+					r.Recorder.Eventf(edge, v1.EventTypeWarning, "Validate", "Could not validate ip restriction: ", err)
+				}
```

Comment:
maybe we should ignore the whole route, and continue with the other ones here 🤔

Reply:
I think that would probably be better. Because all of these steps are done inside this loop over the routes, this could make it have confusing updates where a handful of routes are handled until a bad one errors and skips the rest. 

I think as long as the affected route is not half exposed, trying to keep the rest of the routes up and configure them properly would be the better user experience